stake-pool/js/test/instructions.test.ts

```diff
@@ -253,4 +255,53 @@ describe('StakePoolProgram', () => {
       expect(res.totalRentFreeBalances).toEqual(10000);
     });
   });
+
+  describe('createPoolTokenMetadata', () => {
+    it.only('should create pool token metadata', async () => {
+      connection.getAccountInfo = jest.fn(async (pubKey: PublicKey) => {
+        if (pubKey == stakePoolAddress) {
+          return stakePoolAccount;
+        }
+        return null;
+      });
+
+      const tokenMetadata = new PublicKey(0);
+
+      const res = await createPoolTokenMetadata(
+        connection,
+        stakePoolAddress,
+        tokenMetadata,
+        'test',
+        'TEST',
+        'https://example.com',
```

Comment:
nit: Since you're comparing against these later, it's easier to store them in variables and compare with the variable rather than hardcoding twice